GameServerLib/Items/Shop.cs

```diff
@@ -28,12 +29,8 @@ public bool HandleItemSellRequest(byte slotId)
 
             var sellPrice = i.TotalPrice * i.ItemData.SellBackModifier;
             _owner.Stats.Gold += sellPrice;
-
-            if (i.ItemData.MaxStack > 1)
-            {
-                i.DecrementStackCount();
-            }
-            RemoveItem(i, slotId, i.StackCount);
+            
+            RemoveItem(i, slotId, 1);
```

Comment:
Please unify there to only be a single way to reduce/sell items, either via the `RemoveItem` or `DecrementStackCount`. You can even call it `RemoveItemStacks` if you want, as long as there's only a single way to do effectively the same thing.

This solution is good enough if you get rid of the `DecrementStackCount` function altogether

Reply:
I find both the increment/decrement methods to be not worth keeping, as it only creates ambiguity like this. Do you think it would be better to just remove both `IncrementStackCount` and `DecrementStackCount` from `IStackable`? In all implementations, `IncrementStackCount` is the only one ever used, inside of `Item`.

Reply:
As long as there's a single interface for managing the stack count, I don't hold much preference over what that should be as long as the naming is sane. So feel free to remove the inecrement/decrement functions if you think that's best.